.gitignore

```diff
@@ -6,9 +6,6 @@ bower_components/
 package-lock.json
 yarn.lock
 
-# IDEs and editors
-.idea/
-
```

Comment:
.idea contain already a generated gitignore that manage what can be sync with a team and what shoudn't.